.github/workflows/build_windows.yml

```diff
@@ -0,0 +1,146 @@
+        ref: ${{ inputs.ref }}
+
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.10'
+
+    - name: Cache pip dependencies
+      uses: actions/cache@v4
+      with:
+        path: ~\AppData\Local\pip\Cache
+        key: ${{ runner.os }}-pip-${{ hashFiles('setup.py', 'build_requires.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+
+    - name: Install build dependencies
+      run: make dependencies
+
+    - name: Install wget and Inno Setup
+      run: choco install wget innosetup --no-progress
+
+    - name: Download and install the whl from URL
+      if: ${{ inputs.whl-url }}
+      run: make get-whl whl=${{ inputs.whl-url }}
+
+    - name: Download the whl from artifacts
+      if: ${{ inputs.whl-file-name }}
+      uses: actions/download-artifact@v4
+      with:
+        name: ${{ inputs.whl-file-name }}
+        path: whl
+
+    - name: Install whl from artifacts
+      if: ${{ inputs.whl-file-name }}
+      run: make install-whl whl=whl/${{ inputs.whl-file-name }}
+
+    - name: Build the application with PyInstaller
+      run: make pyinstaller
+
+    - name: Build the Windows installer
+      run: make build-installer-windows
+
+    - name: Get EXE filenames
+      id: get-exe-filename
+      shell: pwsh
+      run: |
+        $baseName = (Get-ChildItem -Path dist-installer -Filter *.exe).BaseName
+        $unsignedName = "$baseName-unsigned.exe"
+        $finalName = "$baseName.exe"
+        # Rename the built file to have the "-unsigned" suffix
+        Rename-Item -Path "dist-installer\$baseName.exe" -NewName $unsignedName
+        echo "unsigned-exe-file-name=$unsignedName" | Out-File -FilePath $env:GITHUB_OUTPUT -Encoding utf8 -Append
+        echo "final-exe-file-name=$finalName" | Out-File -FilePath $env:GITHUB_OUTPUT -Encoding utf8 -Append
+
+    - name: Sign files with Azure Trusted Signing
+      if: ${{ inputs.release }}
+      uses: azure/trusted-signing-action@v0
+      with:
+        azure-tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+        azure-client-id: ${{ secrets.AZURE_CLIENT_ID }}
+        azure-client-secret: ${{ secrets.AZURE_CLIENT_SECRET }}
+        endpoint: 'https://wus2.codesigning.azure.net/'
+        trusted-signing-account-name: 'LE-Trusted-Signing-Acct'
+        certificate-profile-name: 'LE-Windows-Certificates'
+        files-folder: dist-installer
+        files-folder-filter: ${{ steps.get-exe-filename.outputs.unsigned-exe-file-name }}
+        file-digest: SHA256
+        timestamp-rfc3161: 'http://timestamp.acs.microsoft.com'
+        timestamp-digest: 'SHA256'
+
+    - name: Prepare Final Artifact
+      shell: pwsh
+      run: |
+        # Always rename the file from "-unsigned.exe" to its final name before uploading.
+        Rename-Item -Path "dist-installer\${{ steps.get-exe-filename.outputs.unsigned-exe-file-name }}" -NewName "${{ steps.get-exe-filename.outputs.final-exe-file-name }}"
```

Comment:
This seems like a change from the existing behaviour in the old windows installer: https://github.com/learningequality/kolibri-installer-windows/blob/main/.github/workflows/build_exe.yml#L123

Here we specifically retain `unsigned` in the installer name so that it's clear that the asset is not a release asset. I think it's worth retaining this behaviour.